core/templates/dev/head/editor/ExplorationEditor.js

```diff
@@ -759,6 +761,14 @@ oppia.controller('ExplorationSaveAndPublishButtons', [
           }, function() {
             $scope.isSaveInProgress = false;
           });
+
+          if ($scope.isPrivate()) {
+            siteAnalyticsService.registerSaveExplorationDraftEvent(
```

Comment:
After some more thought, changed to CommitChangesToPrivateExplorationEvent, CommitChangesToPublicExplorationEvent respectively. Thanks!